### PR TITLE
try mysql-server in workflow

### PR DESCRIPTION
**Pull request by pengyin-shan**
1. Adjust MySQL installation process for Ubuntu LTS to install both mysql-server and mysql-client. Separating installation commands to avoid dependency problem
2. Separating pdf download tests to avoid multi-process running problem from wkhtmltopdf

### CODE REVIEW

**Review by lagoan — CHANGES_REQUESTED** [review]
All looks good! We just need a an entry on the CHANGELOG file to indicate the changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -165,4 +165,7 @@
  - Fix 500 error when downloading usage statistics
  - Fix name of downloaded usage file
  - Allow organizational admin to see the download button in 'Statisctics on Users' and 'Statistics on Tempalte' accordions
-- Upgrade node-sass to 7, node to 12 and adjust eslint test to use Github default action
+ - Upgrade node-sass to 7, node to 12 and adjust eslint test to use Github default action
+
+## [3.0.4+portage-3.0.0] - 2021-12-09
+  - Update mysql installation process for the Github action
```
Comment (lagoan) [inline CHANGELOG.md:171]: We can improve this change by adding the entry to the top of the file. Something like:

```
## [Unreleased]

### Changed
 - Update mysql installation process for the Github action

```

Sorry for the confusion, I missed adding the changes for version 3.0.0 when I tagged the release.

I will add the changes shortly.

Comment (pengyin-shan) [reply CHANGELOG.md:171]: Got it. I'll change this one to the Unreleased section

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [Unreleased]
+
+## Changed
```
Comment (lagoan) [inline CHANGELOG.md:5]: Very minor issue. This should use three `#` signs instead of two, like:
```
### Changed
```

**Review by lagoan — APPROVED** [review]
LGTM

**Pull request merged**